src/main/java/com/stories/ApiApplication.java

```diff
@@ -1,15 +1,15 @@
-package com.stories;	
-
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-
-@SpringBootApplication
-public class ApiApplication {
-
-	public static void main(String[] args) {
-		SpringApplication.run(ApiApplication.class, args);
-		 
-		
-	}
-
-}
+package com.stories;	
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ApiApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(ApiApplication.class, args);
+		 

```

Comment:
Please remove the extra space and replace "ApiApplication" to "StoriesApplication".